server.js

```diff
@@ -0,0 +1,80 @@
+require('dotenv').config();
+
+// Application Dependencies
+const express = require('express');
+const cors = require('cors');
+
+// Application Setup
+// - make an express app!
+const app = express();
+// - get the port on which to run the server
+const PORT = process.env.PORT;
+
+const geoData = require('./data/geo.json');
+const weatherData = require('./data/darksky.json');
+// - enable CORS
+app.use(cors());
+
+app.get('/location', (request, response) => {
+    try {
+        const location = request.query.location;
+        const result = getLatLng(location);
+        response.status(200).json(result);
+    }
+
+    catch(err) {
+        // TODO: make an object and send via .json...
+        response.status(500).send('Sorry something went wrong, please try again');
+    }
+
+});
+
+app.get('/weather', (request, response) => {
+    try {
+        const weather = request.query.weather;
+        const result = getForecast(weather);
+        response.status(200).json(result);
+    }
+
+    catch(err) {
+        // TODO: make an object and send via .json...
+        response.status(500).send('Sorry something went wrong, please try again');
+    }
+
+});
+
+function getForecast() {
+    const eightDayForecast = [];
+
+    for(let i = 0; i < weatherData.daily.data.length; i++) {
```

Comment:
psstt . you can use a forEach, its like way less writing, and itll make you look for modern.

Reply:
.map would also be great here (vs pushing to a predefined array)

Reply:
got that .map working